demos/include/config.h

```diff
@@ -29,7 +29,7 @@ typedef int MQTTNetworkContext_t;
 #define MQTT_MAX_QUEUED_PUBLISH_MESSAGES    10
 
 /* Include file for POSIX reference implementation. */
-#include "platform/include/iot_logging.h"
+#include "iot_logging.h"
```

Comment:
Since you are chancing this include path do we need to update:
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/platform/posix/iot_logging.c#L42
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/platform/posix/iot_logging.c#L36

Reply:
It builds without them, but yeah I think that makes sense.